lib/ansible/modules/cloud/openstack/os_server_facts.py

```diff
@@ -71,6 +79,8 @@ def main():
     try:
         openstack_servers = cloud.list_servers(
             detailed=module.params['detailed'])
+        openstack_servers = cloud.search_servers(
```

Comment:
Why do we set the openstack_servers variable twice in a row?  Is the first call required?  Seems like the second call will overwrite the result from the first call.